src/main/scala/codacy/rubocop/RubocopResult.scala

```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
 
 case class RubocopLocation(line: Int, column: JsValue, length: JsValue)
 
-case class RubocopMetadata(rubocop_version: JsValue, ruby_engine: JsValue, ruby_version: JsValue, ruby_patchlevel: JsValue, ruby_platform: JsValue)
-
-case class RubocopOffense(severity: JsValue, message: JsString, cop_name: JsString, corrected: JsValue, location: RubocopLocation)
+case class RubocopMetadata(
+    rubocop_version: JsValue,
+    ruby_engine: JsValue,
+    ruby_version: JsValue,
+    ruby_patchlevel: JsValue,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'ruby_patchlevel' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)